.github/workflows/sdk-pr.yaml

```diff
@@ -31,11 +31,19 @@ jobs:
         uses: ./.github/actions/build
         with:
           go-version: ${{ env.GO_VERSION }}
-      - name: Generate SDK
-        uses: ./.github/actions/generate-sdk
-      - name: Push SDK
+      - name: Generate Go SDK
+        uses: ./.github/actions/generate-sdk/go
+      - name: Push Go SDK
         env: 
           GH_REPO: 'stackitcloud/stackit-sdk-go'
           GH_TOKEN: ${{ secrets.SDK_PR_TOKEN }}
         run: |
-          scripts/sdk-create-pr.sh "generator-bot-${{ github.run_id }}" "Generated from GitHub run [${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
\ No newline at end of file
+          scripts/sdk-create-pr.sh "generator-bot-${{ github.run_id }}" "Generated from GitHub run [${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
+      - name: Generate Python SDK
```

Comment:
This should also be a separate job

Reply:
you didn't separate this into two jobs, why? I think the python SDK generation should not fail if Go fails, they are independent

Reply:
You are correct. I split it